### ISSUE TITLE
Permission Denied

### ISSUE THREAD

**Issue by fellipeh** [olipo186/Git-Auto-Deploy#122]
Hi, I trying to install on AWS server, and get these error:

```
$ service git-auto-deploy start
Starting..
start-stop-daemon: unable to open pidfile /var/run/git-auto-deploy/git-auto-deploy.pid (Permission denied)
```

here is:
```
$ sudo ls -la /var/run/git-auto-deploy/
total 0
drwxr-x---  2 git-auto-deploy git-auto-deploy  60 Jul 13 16:45 .
drwxr-xr-x 21 root            root            780 Jul 13 16:45 ..
-rwxr-x---  1 git-auto-deploy git-auto-deploy   0 Jul 13 16:45 git-auto-deploy.pid
```

I run my system with `ubuntu` user, not root... how can I fix that?

**Comment by realsby** [comment]
when your ubuntu startup before run git-auto-deploy change /var/run/git-auto-deploy/ folder permissions.. after that it can create pid files..

**Comment by olipo186** [comment]
Fixed and released in version 0.4 👍
